eth2util/hash.go

```diff
@@ -40,3 +43,14 @@ func SlotHashRoot(slot eth2p0.Slot) ([32]byte, error) {
 
 	return hash, nil
 }
+
+// RandomHex32 returns a random 32 character hex string. It uses crypto/rand.
+func RandomHex32() (string, error) {
```

Comment:
This is used by `cmd`, `dkg` and `keymanager` packages. Although, it is not specifically "eth2" related so suggestions welcome on where to move this to

Reply:
can just duplicate, it is pretty standard stuff

Reply:
I would increase the length of these passwords to 32bits. 64 chars